package.json

```diff
@@ -46,6 +47,7 @@
     "load-grunt-tasks": "1.0.0"
   },
   "dependencies": {
+    "bower": "1.3.12",
```

Comment:
nit: instead of doing this, we also have an option to install the required **node.js environment** dependencies via npm and exclude Bower as a dependency (this way it is not downloaded for those who only use in node.js). We do this in the fxa-js-client.

i.e with p-promise we have it in both locations: https://github.com/mozilla/fxa-js-client/blob/master/package.json#L33 and https://github.com/mozilla/fxa-js-client/blob/master/bower.json#L11
Though this approach might not be a big deal and will help avoid repetition anyway